administrator-manual/en/freepbx.rst

```diff
@@ -2,29 +2,35 @@
 FreePBX
 =======
 
-.. note::
-
-  The configuration page of this module is available only in the old Server Manager.
-
 :index:`FreePBX` is a web-based open source GUI (graphical user interface)
 that controls and manages :index:`Asterisk` (PBX), an open source communication server
 (https://www.freepbx.org/).
 
 Installation
 ============
 
-You can install FreePBX from the package manager of NethServer, the module
+You can install FreePBX from the package manager of |product|, the module
```

Comment:
For nsent FreePBX is bundled with NethVoice and must not be installed separately

Reply:
FreePBX is not visible inside the Enterprise Software Manager.
By the way, if you install it before NethVoice, nothing will break.